datafusion/functions-aggregate/src/string_agg.rs

```diff
@@ -178,6 +178,10 @@ impl AggregateUDFImpl for StringAgg {
         )))
     }
 
+    fn reverse_expr(&self) -> datafusion_expr::ReversedUDAF {
```

Comment:
Can you please add this on `main` _**first**_?
This would make it impossible that anyone updating from 49.0.1 to a newer DF version experiences a behavioral regression.